control-plane-operator/controllers/hostedcontrolplane/assets/olm/catalog-community.cronjob.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: community-operators-catalog
+  annotations:
+    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+spec:
+  concurrencyPolicy: Forbid
+  schedule: '* */1 * * *'
```

Comment:
This could be parameterized with templating. Currently the job will run every hour.